src/styles/global.css

```diff
@@ -48,10 +45,16 @@ html[data-theme="dark"] {
   [role="button"]:not(:disabled) {
     cursor: pointer;
   }
-  section,
-  footer {
-    @apply mx-auto max-w-3xl px-4;
-  }
+}
+
+body {
```

Comment:
Moved body for outside of `Layer base`, I was getting some issues, to make the box work